documentation/common/attributes.adoc

```diff
@@ -53,7 +53,7 @@
 // Other links
 
 // Section enablers
-:Kubernetes:
+:Kubernetes: Yes
```

Comment:
I think just adding the element here enables it? @ncbaratta can you confirm please?

Reply:
Yeah, that was my bad just testing something!